pkgs/top-level/all-packages.nix

```diff
@@ -16906,6 +16906,8 @@ with pkgs;
     buildShareware = true;
   };
 
+  rquickshare-legacy = callPackage ../by-name/rq/rquickshare/package.nix { app-type = "legacy"; };
```

Comment:
I am unsure how to prevent r-ryanthm from trying to update `rquickshare-legacy` instead of `rquickshare`.